koin-projects/examples/androidx-samples/src/test/java/CheckModulesTest.kt

```diff
@@ -0,0 +1,25 @@
+import org.junit.Test
+import org.junit.experimental.categories.Category
+import org.koin.core.parameter.parametersOf
+import org.koin.sample.androidx.components.mvp.FactoryPresenter
+import org.koin.sample.androidx.components.mvp.ScopedPresenter
+import org.koin.sample.androidx.di.appModule
+import org.koin.sample.androidx.di.mvpModule
+import org.koin.sample.androidx.di.mvvmModule
+import org.koin.test.category.CheckModuleTest
+import org.koin.test.check.checkModules
+
+@Category(CheckModuleTest::class)
+class CheckModulesTest {
+
+    @Test
+    fun `test DI modules`() = checkModules(
+            parameters = {
+                create<FactoryPresenter> { parametersOf("_ID_") }
+                create<ScopedPresenter> { parametersOf("_ID_") }
+            }
+    ) {
+        modules(appModule + mvpModule + mvvmModule)// + mvpModule + mvvmModule + scopeModule)
```

Comment:
Remove `mvvmModule` to fix tests:
```suggestion
        modules(appModule + mvpModule)
```

Or provide parameters to dependencies from `mvvmModule`:
```kotlin
create<SimpleViewModel> { parametersOf("_ID_") }
create<SimpleViewModel>(named("vm1")) { parametersOf("_ID_") }
create<SimpleViewModel>(named("vm2")) { parametersOf("_ID_") }
create<SavedStateViewModel> { parametersOf(SavedStateHandle(), "_ID_") }
create<SavedStateViewModel>(named("vm2")) { parametersOf(SavedStateHandle(), "_ID_") }
```

(https://circleci.com/gh/InsertKoinIO/koin/625?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

Reply:
thanks, I was working on it and just push it. Should have pushed it as draft :/